packages/driver/src/cy/location.ts

```diff
@@ -17,8 +31,46 @@ export const create = (state: StateFunc) => ({
     } catch (e) {
       // it is possible we do not have access to the location
       // for example, if the app has redirected to a different origin
-      return ''
+      return $Location.create('')
```

Comment:
I'm now always returning a location object, not sure this is totally correct but also not sure i want to go so far as to throw an error where we haven't been throwing before. All code that used getRemoteLocation expected a location  object to be returned.